### PR TITLE
LPD-17296 Adds bulk permission action for web content admin

### PR DESCRIPTION
**Pull request by jkappler**
https://liferay.atlassian.net/browse/LPD-17296

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
thanks @ealonso! Changes applied

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -111,12 +176,34 @@ export default function propsTransformer({
...
+	) => {
+		const url = new URL(permissionsURL);
+
+		openSelectionModal({
+			title: Liferay.Language.get('permissions'),
+			url: addParams(
+				{
+					[`_${url.searchParams.get(
+						'p_p_id'
+					)}_resourcePrimKey`]: resourcePrimKeys.join(','),
```
Comment (victorg1991) [inline modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ManagementToolbarPropsTransformer.js:191]: Can we use createPortletRequest utility here? (just a minor suggestion)

Comment (jkappler) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ManagementToolbarPropsTransformer.js]: Thanks @victorg1991, changes applied 😻

**Comment by jkappler** [comment]
Moved here: https://github.com/liferay-content-management/liferay-portal/pull/5124

**Pull request closed without merging**